src/main/java/services/notification_sending/EmailNotificationPresenter.java

```diff
@@ -0,0 +1,81 @@
+package services.notification_sending;
+
+import java.util.Properties;
+import javax.mail.*;
+import javax.mail.internet.*;
+
+/**
+ * Sends out email notifications.
+ * Keeps a record of whether the user enabled the email notification settings.
+ */
+public class EmailNotificationPresenter implements NotificationPresenter, SettingsRegistry {
+    private boolean enabled;
+    private String senderEmail;
+    private String password;
+    private String userEmail;
+    private String subject;
+
+    EmailNotificationPresenter(boolean enabled, String senderEmail, String password, String userEmail, String subject) {
+        this.enabled = enabled;
+        this.senderEmail = senderEmail;
+        this.password = password;
+        this.userEmail = userEmail;
+        this.subject = subject;
+    }
+
+    @Override
+    public void presentNotification(String message) {
+        //Get properties object
```

Comment:
This kind of comment is more distracting than it is helpful. Comments should be things that are difficult to express in the code (e.g. why are you using "465" as the port?). If you really feel that a bundle of lines should be categorized, it's better self-documented as a private method. In this scenario, it's also a little more reusable that way because you can then make it into a factory method, taking in stuff like the port or SSL factory or host.